addons/nodot/interaction/Teleport3D.gd

```diff
@@ -0,0 +1,88 @@
+## Teleport3D node
```

Comment:
Need a more revealing description such as `Teleport3D is an Area3D which, when touched, teleports the collider to another location either instantly or after a defined time`